boards/96boards/96b_argonkey/doc/index.rst

```diff
@@ -77,9 +77,8 @@ features:
 More information about the board can be found at the
 `ARGONKEY website`_.
 
-The default board configuration can be found in the defconfig file:
-
-        ``boards/arm/96b_argonkey/96b_argonkey_defconfig``
+The default board configuration can be found in
+:zephyr_file:`boards/96boards/96b_argonkey/96b_argonkey_defconfig`
```

Comment:
Question: Is that:
- a new change requested by the hwmV2
- a change that was needed to be done since longtime but is now breaking with migration
- a required, non breaking, but not hwmv2 related change ?

Reply:
Mix of the last two, it was a feature introduced a long time ago and the benefit is that it can be used in the doc build to check for broken links, unrelated to hwmv2.